server/src/routes/api/v1/tablesRouter.js

```diff
@@ -0,0 +1,38 @@
+import express from "express"
+import cleanUserInput from "../../../services/cleanUserInput.js"
+import { ValidationError } from "objection"
+import { Table } from "../../../models/index.js"
+import findPlayer from "../../../services/findPlayer.js"
+import findSeason from "../../../services/findSeason.js"
+
+const tablesRouter = new express.Router()
+
+tablesRouter.post("/", async (req, res) => {
+  const userId = req.user.id
```

Comment:
we don't need to send the user in the post request. we can access the current user through `req.user`